lessons/Data-Science/measures-of-center/langs/en-us/index.adoc

```diff
@@ -276,66 +191,54 @@ Consider the following three datasets:
-}
+@QandA{
 
-=== Investigate
-@lesson-instruction{
-- Turn to @printable-exercise{summarizing-columns-moc.adoc} @ifproglang{pyret}{and use the code provided} to compute and record the `modes` of the `pounds` column.
-- Then complete the remaining questions in the *Summarizing the `Pounds` Column* section.
-}
+@Q{If you heard that the mean age of students in a kindergarten class was 21, would you be surprised? Why or why not?}
+@A{Sample response: yes, that would be surprising. Usually students in kindergarten are 4 or 5 years old!}
 
-@slidebreak
+@Q{Is the median always one of the values in the dataset? If not, when is it not?}
+@A{No, the median is not always one of the values in the dataset. Sometimes, when there are an even number of datapoints, we need to average the two middle values to find the median.}
 
-@QandA{
-@Q{What did you learn from calculating the mode(s)?}
-@A{The most common animal weights are 0.1 and 6.5! That’s well below our mean and even our median, which is further evidence of outliers or skewness.}
+@Q{How come we can find the mode of a categorical dataset, but not the median or the mean?}
+{Finding the mode does not require us to perform any arithmetic computations. Computing the median or the mean does require us to perform some arithmetic, therefore we can only use quantitative data.}
 
-@Q{Can we find the mean, median and mode(s) for any column?}
-@A{No! We can only calculate Measures of Center for @vocab{quantitative} columns.} 
-@A{Note: Not all columns that contain numbers are quantitative! Taking the average of a list of zip codes doesn’t tell us anything at all!}
 }
 
-=== Synthesize
 
-- What must be true about a dataset for the mode(s) to do a good job of describing what is typical?
-- What can we learn from the modes of a dataset?
+== Choosing the Right Measure of Center @duration{15 minutes}
 
-== The Risk of Summarizing Data with a Single Number @duration{15 minutes}
 
 === Overview
 
-Students consider the complexity of summarizing with a single number and learn how to decide which measure of center to use when. They then choose a column, compute all of its measures of center in @proglang, and interpret the results. Finally, they practice computing measures of center for a small dataset by hand and use their findings to critique misleading statements.
+Students use @proglang to compute measure of center, and then consider which measure of center is most appropriate in a given situation.
 
 === Launch
 
-Summarizing a big dataset means that some information gets lost, so it’s important to pick an appropriate summary. Picking the wrong summary can have serious implications! 
+
+Summarizing a big dataset means that some information gets lost, so it's important to pick an appropriate summary (mean, median, or mode). Picking the wrong summary value can have serious implications!
```

Comment:
This is an opportunity to bring kids back to the opening prompt, where they found the "typical" value. Now you can ask "so there are at least three different ways to measure center - some of which you used! - but which one is RIGHT one?"

You could even give the class a minute to discuss beforehand, to gauge what they know

Reply:
Hm. Curiuos how attached you are to changing the opening prompt. LMK. If you feel strongly, I will update this, too.